internal/utils/vuln_hash/vuln_hash.go

```diff
@@ -28,21 +28,37 @@ import (
 //
 // nolint:funlen
 func Bind(vuln *vulnerability.Vulnerability) *vulnerability.Vulnerability {
+	// It generates the hash without the commitEmail and description information, avoiding frequent hash changes
+	// every time that this data changes. Below two more hashes are generated, these contain two old versions of
+	// hashes that are still valid. After the depreciation, only this hash generated with row, column and code
+	// will be considered valid.
 	vuln.VulnHash = crypto.GenerateSHA256(
 		toOneLine(vuln.Code),
 		vuln.Line,
-		vuln.Details,
+
+		// The filepath should be relative to avoid generating different hashes between environments
```

Comment:
I think that this comment should be added on Bind function docs?

```go
// Bind create a sha256 hash of the vulnerability using the vulnerability
// code, line and file. The file path should be relative to avoid generating 
// different hashes between environments.
//
// Bind automatically set the Vulnerability.VulnHash field with the generated
// vulnerability hash.
```

What do you think?

Reply:
I noticed that Bind already have a godoc, but I think that don't have such information like the example above.